tests/src/cookbook_numbers1_test.cpp

```diff
@@ -38,19 +40,21 @@ namespace daw::json {
 #ifdef DAW_JSON_CNTTP_JSON_NAME
 		using type =
 		  json_member_list<json_number<"member0">, json_number<"member1">,
-		                   json_number<"member2">, json_number<"member3">>;
+		                   json_number<"member2">, json_number<"member3", json_number<"member4">>;
```

Comment:
This is missing a > after "member3"

Reply:
ah, sorry, that was in the not compiled branch when I tested. fixed!